app/views/lobby/bits.scala

```diff
@@ -160,3 +96,36 @@ object bits:
           if e.isNow then trans.site.eventInProgress() else momentFromNow(e.startsAt)
       )
     )
+
+  def recentTopics(forumTopics: List[lila.forum.RecentTopic])(using Context) =
+    div(cls := "topic-list")(
+      frag(
+        forumTopics
+          .map: topic =>
+            val mostRecent = topic.posts.last
+            div(
+              span(
+                a(
+                  href  := routes.ForumCateg.show(topic.categId),
+                  title := topic.categId.value.split("-").map(_.capitalize).mkString(" "),
```

Comment:
pretty sure forum categ ids are not meant to be converted to human readable text. We shouldn't do that with urlized (slug) text, even less when it can be random, like categId